demo/summer-2016.html

```diff
@@ -215,10 +216,10 @@
 <script src="http://maxcdn.bootstrapcdn.com/bootstrap/2.3.2/js/bootstrap.min.js"></script>
 
 <script>
+// var editor;
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/katex-previewer", "ace/ext/papeeria/highlighter"], 
-  function(ace, langTools, katexPreviewer, highlighter) {
-    var editor = ace.edit("editor");
+require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/context-previewer", "ace/ext/papeeria/highlighter"], function(ace, langTools, katexPreviewer, highlighter) {
+    editor = ace.edit("editor");
```

Comment:
Is it necessary? It's kind of handy to have this variable in global scope in browser, that lets me experiment right in the browser console.